packages/typescript/src/symbols/reference.tsx

```diff
@@ -32,7 +34,7 @@ export function ref(
 
   return memo(() => {
     if (resolveResult.value === undefined) {
-      return ["<Unresolved Symbol>", undefined];
+      return [`<Unresolved Symbol: ${getRefkeyString(refkey)}>`, undefined];
```

Comment:
as every language needs this, should core provide an helper for that piece of code or do we see some language wanting a specific syntax so maybe it doesn't fail compilation(if thats ever wanted)